### PR TITLE
fix/docs: Fix the broken doc links

### PR DESCRIPTION
**Pull request by Aisuko**
# What does this PR do?

This PR fixed the broken document address in `write_own_pipline.mdx` file. And it was already checked locally by using Huggingface doc-builder. And v0.17.x and v0.16.x version of document has the same issue too.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by pcuenca — COMMENTED** [review]
Thanks a lot! Can you try to commit the first link without the relative path? I think it should work :)

File: docs/source/en/using-diffusers/write_own_pipeline.mdx [hunk 1]
```diff
@@ -286,5 +286,5 @@ This is really what 🧨 Diffusers is designed for: to make it intuitive and eas
 
 For your next steps, feel free to:
 
-* Learn how to [build and contribute a pipeline](using-diffusers/#contribute_pipeline) to 🧨 Diffusers. We can't wait and see what you'll come up with!
-* Explore [existing pipelines](./api/pipelines/overview) in the library, and see if you can deconstruct and build a pipeline from scratch using the models and schedulers separately.
+* Learn how to [build and contribute a pipeline](../using-diffusers/contribute_pipeline) to 🧨 Diffusers. We can't wait and see what you'll come up with!
```
[the remark this answers is not in the document]
Comment (Aisuko) [reply docs/source/en/using-diffusers/write_own_pipeline.mdx]: Let's check the `HuggingFaceDocBuilderDev` later

[the remark this answers is not in the document]
Comment (Aisuko) [reply docs/source/en/using-diffusers/write_own_pipeline.mdx]: It looks like `HuggingFaceDocBuilderDev` has been not triggered by the latest commit. I will check it manually.

[the remark this answers is not in the document]
Comment (Aisuko) [reply docs/source/en/using-diffusers/write_own_pipeline.mdx]: I tested it at local environment, we are good to go. @pcuenca

**Review by pcuenca — APPROVED** [review]
Awesome, thanks!

**Pull request merged**